app/src/main/java/com/github/polybooks/core/database/interfaces/BookDatabase.kt

```diff
@@ -55,10 +55,10 @@ interface BookQuery : Query<Book>, Serializable {
     fun searchByTitle(title : String) : BookQuery
 
     /**
-     * Set this query to get the book associated with the given isbn13, if it exists.
+     * Set this query to get the books associated with the given ISBNs, if they exist.
      * (ignoring other filters)
      * */
-    fun searchByISBN13(isbn13: String) : BookQuery
```

Comment:
For compatiblity I'd suggest to still keep this function allowing to search using a string parameter. You can here already write the implementation using the other function : 
```
fun searchByISBN(isbn : String) : BookQuery = {
    searchByISBN(setOf(isbn))
}
```